profiles/2017-03-09/compute/mgmt/compute/models.go

```diff
@@ -51,12 +51,20 @@ const (
 	ReadWrite CachingTypes = original.ReadWrite
 )
 
+func PossibleCachingTypesValues() []CachingTypes {
```

Comment:
maybe add in the readme this new feature ?

Reply:
Good point we've had this for so long in latest I forgot about it.